README.md

```diff
@@ -154,6 +154,14 @@ If you want `response.url` to be reliable, you'll want to set this header. The
 day that you ditch this polyfill and use native fetch only, you can remove the
 header hack.
 
+### Webpack usage
+
+To use fetch with webpack, you will need `import-loader`, `export-loader`, `es6-promise` and require it this way:
```

Comment:
Loader import-loader || export-loader' is not in the npm registry, name must be imports-loader exports-loader .